dashboard/app/controllers/levels_controller.rb

```diff
@@ -8,8 +8,7 @@ class LevelsController < ApplicationController
   include LevelsHelper
   include ActiveSupport::Inflector
   before_action :authenticate_user!, except: [:show, :embed_level, :get_rubric]
-  before_action :require_levelbuilder_mode, except: [:show, :embed_level, :get_rubric, :get_filtered_levels]
-  before_action :require_levelbuilder_mode_or_test_env, only: [:get_filtered_levels]
+  before_action :require_levelbuilder_mode_or_test_env, except: [:show, :embed_level, :get_rubric]
```

Comment:
Why remove `get_filtered_levels`?

Reply:
I removed it from line 11 so that I could get rid of line 12, which became redundant:
```
before_action :require_levelbuilder_mode_or_test_env, except: [:show, :embed_level, :get_rubric, :get_filtered_levels]
before_action :require_levelbuilder_mode_or_test_env, only: [:get_filtered_levels]
```